frontend/src/metabase/components/form/widgets/FormInputWidget.jsx

```diff
@@ -25,24 +26,28 @@ const FormInputWidget = forwardRef(function FormInputWidget(
     autoFocus,
     helperText,
     tabIndex,
+    subtitle,
   },
   ref,
 ) {
   return (
-    <Input
-      {...formDomOnlyProps(field)}
-      type={type}
-      placeholder={placeholder}
-      aria-labelledby={`${field.name}-label`}
-      readOnly={readOnly}
-      autoFocus={autoFocus}
-      error={field.visited && !field.active && field.error != null}
-      rightIcon={helperText && "info"}
-      rightIconTooltip={helperText}
-      tabIndex={tabIndex}
-      fullWidth
-      ref={ref}
-    />
+    <div>
```

Comment:
It's not clear why there is a `div`. Could you explain?

Reply:
Good catch, that's definitely not meant to be there.